### PR TITLE
Remove line breaks from param descriptions

### PR DESCRIPTION
**Pull request by ivov**
Long paragraphs and descriptions containing list items checked in UI.

Related to #2340

### CODE REVIEW

File: packages/design-system/src/components/N8nInputLabel/InputLabel.vue [hunk 1]
```diff
@@ -135,5 +135,9 @@ export default {
 
 .tooltipPopper {
 	max-width: 400px;
+
+	li {
+		margin-left: var(--spacing-s);
+	}
```
Comment (ivov) [inline packages/design-system/src/components/N8nInputLabel/InputLabel.vue:141]: image
image

File: packages/nodes-base/nodes/TimescaleDb/TimescaleDb.node.ts [hunk 2]
```diff
@@ -247,11 +247,7 @@ export class TimescaleDb implements INodeType {
 							},
 						],
 						default: 'multiple',
-						description: [
-							'The way queries should be sent to database.',
-							'Can be used in conjunction with <b>Continue on Fail</b>.',
-							'See the docs for more examples',
-						].join('<br>'),
+						description: 'The way queries should be sent to database. Can be used in conjunction with <b>Continue on Fail</b>. See the docs for more examples',
```
Comment (mutdmour) [inline packages/nodes-base/nodes/TimescaleDb/TimescaleDb.node.ts:250]: link to docs maybe?

File: packages/nodes-base/nodes/Salesforce/TaskDescription.ts [hunk 3]
```diff
@@ -727,10 +704,7 @@ export const taskFields = [
 						value: 'RecurrenceRegenerated',
 					},
 				],
-				description: `Represents what triggers a repeating task to repeat.<br/>
-				 Add this field to a page layout together with the RecurrenceInterval field,
-				  which determines the number of days between the triggering date (due date or close date)
-				  and the due date of the next repeating task in the series.Label is Repeat This Task.`,
+				description: `Represents what triggers a repeating task to repeat. Add this field to a page layout together with the RecurrenceInterval field, which determines the number of days between the triggering date (due date or close date) and the due date of the next repeating task in the series.Label is Repeat This Task.`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Salesforce/TaskDescription.ts:707]:
```suggestion
				description: `Represents what triggers a repeating task to repeat. Add this field to a page layout together with the RecurrenceInterval field, which determines the number of days between the triggering date (due date or close date) and the due date of the next repeating task in the series. Label is Repeat This Task.`,
```

File: packages/nodes-base/nodes/QuickBase/RecordDescription.ts [hunk 4]
```diff
@@ -527,9 +525,7 @@ export const recordFields = [
 			},
 		},
 		default: '',
-		description: `You're updating records in a Quick Base table with data from an external file. In order for a merge like this to work,<br>
-		Quick Base needs a way to match records in the source data with corresponding records in the destination table. You make this possible by<br>
-		choosing the field in the app table that holds unique matching values. This is called a merge field.`,
+		description: `You're updating records in a Quick Base table with data from an external file. In order for a merge like this to work, Quick Base needs a way to match records in the source data with corresponding records in the destination table. You make this possible by choosing the field in the app table that holds unique matching values. This is called a merge field.`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/QuickBase/RecordDescription.ts:528]:
```suggestion
		description: `<p>You're updating records in a Quick Base table with data from an external file. In order for a merge like this to work, Quick Base needs a way to match records in the source data with corresponding records in the destination table. </p><p>You make this possible by choosing the field in the app table that holds unique matching values. This is called a merge field.</p>`,
```

Comment (mutdmour) [reply packages/nodes-base/nodes/QuickBase/RecordDescription.ts:528]: weird tooltip that starts with `You`. not sure how to rephrase this

File: packages/nodes-base/nodes/QuestDb/QuestDb.node.ts [hunk 5]
```diff
@@ -170,11 +170,7 @@ export class QuestDb implements INodeType {
 							},
 						],
 						default: 'independently',
-						description: [
-							'The way queries should be sent to database.',
-							'Can be used in conjunction with <b>Continue on Fail</b>.',
-							'See the docs for more examples',
-						].join('<br>'),
+						description: 'The way queries should be sent to database. Can be used in conjunction with <b>Continue on Fail</b>. See the docs for more examples',
```
Comment (mutdmour) [inline packages/nodes-base/nodes/QuestDb/QuestDb.node.ts:173]:
```suggestion
						description: 'The way queries should be sent to database. Can be used in conjunction with <b>Continue on Fail</b>. See <a href="https://docs.n8n.io/nodes/n8n-nodes-base.questDb/">the docs</a> for more examples',
```

File: packages/nodes-base/nodes/Pushover/Pushover.node.ts [hunk 6]
```diff
@@ -89,8 +89,7 @@ export class Pushover implements INodeType {
 					},
 				},
 				default: '',
-				description: `The user/group key (not e-mail address) of your user (or you),<br>
-				viewable when logged into our <a href="https://pushover.net/">dashboard</a> (often referred to as USER_KEY in our <a href="https://support.pushover.net/i44-example-code-and-pushover-libraries"></a> and code examples)`,
+				description: `The user/group key (not e-mail address) of your user (or you), viewable when logged into our <a href="https://pushover.net/">dashboard</a> (often referred to as USER_KEY in our <a href="https://support.pushover.net/i44-example-code-and-pushover-libraries"></a> and code examples)`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Pushover/Pushover.node.ts:92]:
```suggestion
				description: `The user/group key (not e-mail address) of your user (or you), viewable when logged into our <a href="https://pushover.net/">dashboard</a> (often referred to as USER_KEY in our <a href="https://support.pushover.net/i44-example-code-and-pushover-libraries">libraries</a> and code examples)`,
```

File: packages/nodes-base/nodes/Postgres/Postgres.node.ts [hunk 7]
```diff
@@ -226,11 +226,7 @@ export class Postgres implements INodeType {
 							},
 						],
 						default: 'multiple',
-						description: [
-							'The way queries should be sent to database.',
-							'Can be used in conjunction with <b>Continue on Fail</b>.',
-							'See the docs for more examples',
-						].join('<br>'),
+						description: 'The way queries should be sent to database. Can be used in conjunction with <b>Continue on Fail</b>. See the docs for more examples.',
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Postgres/Postgres.node.ts:229]:
```suggestion
						description: 'The way queries should be sent to database. Can be used in conjunction with <b>Continue on Fail</b>. See <a href="https://docs.n8n.io/nodes/n8n-nodes-base.postgres/">the docs</a> for more examples.',
```

File: packages/nodes-base/nodes/Line/NotificationDescription.ts [hunk 8]
```diff
@@ -136,8 +136,7 @@ export const notificationFields = [
 				name: 'notificationDisabled',
 				type: 'boolean',
 				default: false,
-				description: `true: The user doesn't receive a push notification when the message is sent.<br>
-				false: The user receives a push notification when the message is sent`,
+				description: `true: The user doesn't receive a push notification when the message is sent. false: The user receives a push notification when the message is sent`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Line/NotificationDescription.ts:139]: This format requires multiple lines or else we should rephrase it

File: packages/nodes-base/nodes/Google/Gmail/MessageDescription.ts [hunk 9]
```diff
@@ -244,8 +244,7 @@ export const messageFields = [
 								name: 'property',
 								type: 'string',
 								default: '',
-								description: `Name of the binary property containing the data to be added to the email as an attachment.</br>
-								Multiples can be set separated by comma.`,
+								description: `Name of the binary property containing the data to be added to the email as an attachment. Multiples can be set separated by comma.`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Google/Gmail/MessageDescription.ts:247]:
```suggestion
								description: `Name of the binary property containing the data to be added to the email as an attachment. Multiple properties can be set separated by comma.`,
```

File: packages/nodes-base/nodes/HttpRequest/HttpRequest.node.ts [hunk 10]
```diff
@@ -436,9 +436,7 @@ export class HttpRequest implements INodeType {
 						],
 					},
 				},
-				description: `Name of the binary property which contains the data for the file to be uploaded.<br />
-							For Form-Data Multipart, multiple can be provided in the format:<br />
-							"sendKey1:binaryProperty1,sendKey2:binaryProperty2`,
+				description: `Name of the binary property which contains the data for the file to be uploaded. For Form-Data Multipart, multiple can be provided in the format: "sendKey1:binaryProperty1,sendKey2:binaryProperty2`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/HttpRequest/HttpRequest.node.ts:439]:
```suggestion
				description: `Name of the binary property which contains the data for the file to be uploaded. For Form-Data Multipart, they can be provided in the format: "sendKey1:binaryProperty1,sendKey2:binaryProperty2`,
```

File: packages/nodes-base/nodes/Hubspot/CompanyDescription.ts [hunk 11]
```diff
@@ -896,10 +896,7 @@ export const companyFields = [
 					loadOptionsMethod: 'getCompanyProperties',
 				},
 				default: '',
-				description: `Used to include specific company properties in the results.<br/>
-				By default, the results will only include company ID and will not include the values for any properties for your companys.<br/>
-				Including this parameter will include the data for the specified property in the results.<br/>
-				You can include this parameter multiple times to request multiple properties separed by ,.`,
+				description: `Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your companys. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Hubspot/CompanyDescription.ts:899]:
```suggestion
				description: `<p>Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your company.</p> <p>Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by <code>,</code>.</p>`,
```

File: packages/nodes-base/nodes/Hubspot/CompanyDescription.ts [hunk 12]
```diff
@@ -1101,10 +1097,7 @@ export const companyFields = [
...
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Hubspot/CompanyDescription.ts:1100]:
```suggestion
				description: `<p>Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your company</p>. <p>Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by <code>,</code>.</p>`,
```

File: packages/nodes-base/nodes/Hubspot/CompanyDescription.ts (hunk 11 above)
Comment (mutdmour) [reply packages/nodes-base/nodes/Hubspot/CompanyDescription.ts:899]: What was meant by `companys`? I assume they meant `company` not `companies` or `company's`

File: packages/nodes-base/nodes/Hubspot/ContactDescription.ts [hunk 13]
```diff
@@ -397,10 +397,7 @@ export const contactFields = [
 					},
 				},
 				default: '',
-				description: `Used to include specific company properties in the results.<br/>
-				By default, the results will only include company ID and will not include the values for any properties for your companys.<br/>
-				Including this parameter will include the data for the specified property in the results.<br/>
-				You can include this parameter multiple times to request multiple properties separed by ,.`,
+				description: `Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your companys. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Hubspot/ContactDescription.ts:400]: this file has all the same issues as in the other Hubspot comments. I wonder if `<code>,</code>` or `','` is better here

File: packages/nodes-base/nodes/Hubspot/DealDescription.ts [hunk 14]
```diff
@@ -452,10 +451,7 @@ export const dealFields = [
 					loadOptionsMethod: 'getDealProperties',
 				},
 				default: [],
-				description: `Used to include specific deal properties in the results.<br/>
-				By default, the results will only include Deal ID and will not include the values for any properties for your Deals.<br/>
-				Including this parameter will include the data for the specified property in the results.<br/>
-				You can include this parameter multiple times to request multiple properties separed by ,.`,
+				description: `Used to include specific deal properties in the results. By default, the results will only include Deal ID and will not include the values for any properties for your Deals. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Hubspot/DealDescription.ts:454]: same issues here

File: packages/nodes-base/nodes/Kitemaker/descriptions/SpaceDescription.ts [hunk 15]
```diff
@@ -13,7 +13,7 @@ export const spaceOperations = [
 			{
 				name: 'Get All',
 				value: 'getAll',
-				description: 'Retrieve data on all the spaces in the<br>logged-in user\'s organization.',
+				description: 'Retrieve data on all the spaces in the logged-in user\'s organization.',
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Kitemaker/descriptions/SpaceDescription.ts:16]:
```suggestion
				description: `Retrieve data on all the spaces in the logged-in user's organization.`,
```

File: packages/nodes-base/nodes/Kitemaker/descriptions/UserDescription.ts [hunk 16]
```diff
@@ -13,7 +13,7 @@ export const userOperations = [
 			{
 				name: 'Get All',
 				value: 'getAll',
-				description: 'Retrieve data on all the users in the<br>logged-in user\'s organization.',
+				description: 'Retrieve data on all the users in the logged-in user\'s organization.',
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Kitemaker/descriptions/UserDescription.ts:16]:
```suggestion
				description: `Retrieve data on all the users in the logged-in user's organization.`,
```

File: packages/nodes-base/nodes/Keap/ContactDescription.ts [hunk 17]
```diff
@@ -72,8 +72,7 @@ export const contactFields = [
 			},
 		},
 		default: 'email',
-		description: `Performs duplicate checking by one of the following options: Email, EmailAndName,</br>
-		if a match is found using the option provided, the existing contact will be updated`,
+		description: `Performs duplicate checking by one of the following options: Email, EmailAndName, if a match is found using the option provided, the existing contact will be updated`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Keap/ContactDescription.ts:75]:
```suggestion
		description: `Performs duplicate checking by one of the following options: Email, EmailAndName. If a match is found using the option provided, the existing contact will be updated`,
```

File: packages/nodes-base/nodes/Jira/IssueDescription.ts [hunk 18]
```diff
@@ -550,52 +550,29 @@ export const issueFields = [
...
-				schema Returns the schema describing a field type.<br/>
-				transitions Returns all possible transitions for the issue.<br/>
-				editmeta Returns information about how each field can be edited.<br/>
-				changelog Returns a list of recent updates to an issue, sorted by date, starting from the most recent.<br/>
-				versionedRepresentations Returns a JSON array for each version of a field's value, with the highest number
+				description: `Use expand to include additional information about the issues in the response. This parameter accepts a comma-separated list. Expand options include: <code>renderedFields</code> Returns field values rendered in HTML format. <code>names</code> Returns the display name of each field. <code>schema</code> Returns the schema describing a field type. <code>transitions</code> Returns all possible transitions for the issue. <code>editmeta</code> Returns information about how each field can be edited. <code>changelog</code> Returns a list of recent updates to an issue, sorted by date, starting from the most recent. <code>versionedRepresentations</code> Returns a JSON array for each version of a field's value, with the highest number
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Jira/IssueDescription.ts:553]: not really readable. we need to break this.
Screen Shot 2021-11-24 at 12 49 38 PM

File: packages/nodes-base/nodes/Hubspot/TicketDescription.ts [hunk 19]
```diff
@@ -518,10 +512,7 @@ export const ticketFields = [
 					loadOptionsMethod: 'getTicketProperties',
 				},
 				default: [],
-				description: `Used to include specific ticket properties in the results.<br/>
-			By default, the results will only include ticket ID and will not include the values for any properties for your tickets.<br/>
-			Including this parameter will include the data for the specified property in the results.<br/>
-			You can include this parameter multiple times to request multiple properties separed by ,.`,
+				description: `Used to include specific ticket properties in the results. By default, the results will only include ticket ID and will not include the values for any properties for your tickets. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Hubspot/TicketDescription.ts:515]: same as other hubspot issues

**Review by mutdmour — COMMENTED** [review]
nice job

File: packages/nodes-base/nodes/Hubspot/CompanyDescription.ts (hunk 11 above)
Comment (ivov) [reply packages/nodes-base/nodes/Hubspot/CompanyDescription.ts]: Other params here have the same wording but mention `deals` and `tickets` so it's plural.

File: packages/nodes-base/nodes/QuickBase/RecordDescription.ts (hunk 4 above)
Comment (ivov) [reply packages/nodes-base/nodes/QuickBase/RecordDescription.ts]: Multiline fixed.

Agree but we probably should not dwell on phrasing in this PR. If you like I can rig the nodelinter to extract all param descriptions for someone to review.

File: packages/nodes-base/nodes/Wise/descriptions/TransferDescription.ts [hunk 20]
```diff
@@ -227,7 +227,7 @@ export const transferFields = [
 		type: 'boolean',
 		required: true,
 		default: false,
-		description: 'Download the transfer receipt as a PDF file.<br>Only for executed transfers, having status \'Outgoing Payment Sent\'.',
+		description: 'Download the transfer receipt as a PDF file. Only for executed transfers, having status \'Outgoing Payment Sent\'.',
```
[the remark this answers is not in the document]
Comment (ivov) [reply packages/nodes-base/nodes/Wise/descriptions/TransferDescription.ts:230]: Issue opened: https://github.com/n8n-io/nodelinter/issues/51

File: packages/nodes-base/nodes/Coda/TableDescription.ts [hunk 21]
```diff
@@ -378,9 +376,7 @@ export const tableFields = [
 					alwaysOpenEditWindow: true,
 				},
 				default: '',
-				description: `Query used to filter returned rows, specified as <column_id_or_name>:<value>. <br/>
-				If you'd like to use a column name instead of an ID, you must quote it (e.g., "My Column":123).<br/>
-				Also note that value is a JSON value; if you'd like to use a string, you must surround it in quotes (e.g., "groceries").`,
+				description: `Query used to filter returned rows, specified as <column_id_or_name>:<value>. If you'd like to use a column name instead of an ID, you must quote it (e.g., "My Column":123). Also note that value is a JSON value; if you'd like to use a string, you must surround it in quotes (e.g., "groceries").`,
```
[the remark this answers is not in the document]
Comment (ivov) [reply packages/nodes-base/nodes/Coda/TableDescription.ts:379]: Issue opened: https://github.com/n8n-io/nodelinter/issues/52

File: packages/nodes-base/nodes/Hubspot/CompanyDescription.ts [hunk 22]
```diff
@@ -896,15 +896,14 @@ export const companyFields = [
...
+				description: `<p>Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your companies.</p><p>Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by a comma: <code>,</code>.</p>`,
 			},
 			{
 				displayName: 'Properties With History',
 				name: 'propertiesWithHistory',
 				type: 'string',
 				default: '',
-				description: `Works similarly to properties=, but this parameter will include the history for the specified property,
-				instead of just including the current value. Use this parameter when you need the full history of changes to a property's value.`,
+				description: `Works similarly to properties=, but this parameter will include the history for the specified property, instead of just including the current value. Use this parameter when you need the full history of changes to a property's value.`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Hubspot/CompanyDescription.ts:906]: `properties=` is this a typo or does it mean something? not familiar enough with hubspot to know

File: packages/nodes-base/nodes/Hubspot/DealDescription.ts [hunk 23]
```diff
@@ -778,7 +778,7 @@ export const dealFields = [
 					loadOptionsMethod: 'getDealProperties',
 				},
 				default: [],
-				description: `Used to include specific deal properties in the results. By default, the results will only include Deal ID and will not include the values for any properties for your companys. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
+				description: `<p>Used to include specific deal properties in the results. By default, the results will only include company ID and will not include the values for any properties for your company.</p><p>Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by a comma: <code>,</code>.</p>`,
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Hubspot/DealDescription.ts:781]: was the change from `Deal` to `company` intentional? not familiar enough with this node to know

**Review by mutdmour — APPROVED** [review]
LGTM. Just a couple of questions to clarify

File: packages/nodes-base/nodes/Hubspot/CompanyDescription.ts (hunk 22 above)
Comment (ivov) [reply packages/nodes-base/nodes/Hubspot/CompanyDescription.ts:906]: image
https://legacydocs.hubspot.com/docs/methods/deals/get-associated-deals

File: packages/nodes-base/nodes/Hubspot/DealDescription.ts (hunk 23 above)
Comment (ivov) [reply packages/nodes-base/nodes/Hubspot/DealDescription.ts]: Copy-paste artifact

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.151.0`

**Pull request merged**